fiftyone/core/annotation/generate_label_schemas.py

```
@@ -5,6 +5,7 @@
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
+from typing import Iterable, Literal, Optional, TYPE_CHECKING, Union
```

Ruff (0.14.14)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)